SPECS/bpftrace/bpftrace-0.20-llvm18.patch

```diff
@@ -0,0 +1,515 @@
+From 686b73cbb7ae5ab24d1f8ad69c608225d14c94e2 Mon Sep 17 00:00:00 2001
```

Comment:
Personally, I prefer if our patch file commit ids actually match the upstream commit id (makes it much easier to compare against the actual upstream commits). For example this patch upstream has commit id ce9a456d184d9aac21b2fefcbd46212391b6ebea